### PR TITLE
MFTF: Extract Action Groups to separate files for dev/tests

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Extract each Action Group to separate file, to follow MFTF Best Practices.

### Fixed Issues (if relevant)
1. https://github.com/magento/magento2/issues/26331

### Questions or comments
Had to extract the changes per-module. The previous try failed, because of conflicts.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6565](https://jira.corp.magento.com/browse/ENGCOM-6565) has been created to process this Pull Request

File: dev/tests/acceptance/tests/functional/Magento/FunctionalTest/ConfigurableProductWishlist/ActionGroup/StorefrontCustomerCheckConfigurableProductInWishlistActionGroup.xml [hunk 1]
```diff
@@ -24,20 +24,4 @@
         <seeElement selector="{{StorefrontCustomerWishlistProductSection.ProductAddToCartByName(productVar.name)}}" stepKey="AssertWishlistAddToCart"/>
         <seeElement selector="{{StorefrontCustomerWishlistProductSection.ProductImageByName(productVar.name)}}" stepKey="AssertWishlistProductImage"/>
     </actionGroup>
-
-    <!-- Check configurable product in wishlist sidebar -->
-    <actionGroup name="StorefrontCustomerCheckConfigurableProductInWishlistSidebar">
```
Comment (dmytro-ch) [inline dev/tests/acceptance/tests/functional/Magento/FunctionalTest/ConfigurableProductWishlist/ActionGroup/StorefrontCustomerCheckConfigurableProductInWishlistActionGroup.xml]: Shouldn't we keep this action group in a separate file, but marked as deprecated for backward compatibility?
@lbajsarowicz, what do you think?

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

File: dev/tests/acceptance/tests/functional/Magento/FunctionalTest/ConfigurableProductWishlist/ActionGroup/StorefrontCustomerCheckConfigurableProductInWishlistActionGroup.xml (hunk 1 above)
Comment (lbajsarowicz) [reply dev/tests/acceptance/tests/functional/Magento/FunctionalTest/ConfigurableProductWishlist/ActionGroup/StorefrontCustomerCheckConfigurableProductInWishlistActionGroup.xml]: We should, actually.

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6565](https://jira.corp.magento.com/browse/ENGCOM-6565) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Pull request merged**